WooCommerce/Classes/ViewRelated/Products/ProductSelector/TopProductsFromCachedOrdersProvider.swift

```diff
@@ -0,0 +1,80 @@
+import Foundation
+import Yosemite
+import Storage
+
+/// Encapsulates the ids of those products that were most or last sold among the cached orders
+///
+struct TopProductsFromCachedOrders: Equatable {
+    let popularProductsIds: [Int64]
+    let lastSoldProductsIds: [Int64]
```

Comment:
Thanks for the comment here @iamgabrielma 

I see your point, and I also considered that. But since these will be the name of the sections, I preferred the leave it like that, to prioritize consistency along the app. That way we can find where these elements are found. See pdcxQM-223-p2#comment-1868 also about the discussion on last Sold - recently sold (recently implies a frame of recent time, while these products are not necessarily sold recently).